tests/test_fellow.py

```diff
@@ -0,0 +1,54 @@
+import unittest
+
+from models.fellow import Fellow
+
+
+class TestFellow(unittest.TestCase):
+    """Test cases for the Fellow class"""
+
+    def setUp(self):
+        """Setup a new Fellow object to be used for testing"""
+        self.fellow = Fellow('Brian', 'Mosigisi')
+
+    def test_fellow_type(self):
+        """Tests if the object created is of type Fellow"""
+        self.assertEqual(type(self.fellow), Fellow)
```

Comment:
under what circumstances would you instantiate `Fellow` and that instance not be an instance of `Fellow`?